### DISCUSSION TITLE
Display Dynamic Product "Reference" Number

### DISCUSSION THREAD

**Discussion by Yan-Studio-WEB** [PrestaShop/PrestaShop#31664] [Ideas]
### The problem

When I have a Product with combinations, Presta **give me** the ability to add for each combination a unique reference number. But i can t see it in front end when i click on combinations.

### The solution

I believe will be nice to add the function to display corresponding reference number for each combination.
I'm not a programmer but I hope someday somebody will fix that because a lot of my clients ask for it

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by SharakPL** [comment]
One of my clients has just asked about it too. Should be easy to implement in the theme. I'm on it...

**Reply by Yan-Studio-WEB** [reply]
Yes please do that this have to be a default behavior please let me know for your progress
Thanks

**Reply by SharakPL** [reply]
Turns out it already updates product reference on combination change :)
Tested on 1.7.8.8 and 8.1.0 (develop).

https://user-images.githubusercontent.com/5007456/224187938-07aa64dc-d99f-40af-baf1-b8b41c3e621d.mp4

**Comment by eternoendless** [comment]
It appears that this has been done already :)
